app/styles/_library/_config.imports.scss

```diff
@@ -127,7 +127,7 @@ $tests: false;
 @import "nypr-m-toggle-box";
 @import "nypr-o-persistent-audio-player";
 @import "nypr-m-play-button";
-@import "nypr-o-main-audio-player";
+@import "./components/whats-on";
```

Comment:
When you rebase to pull in the button branch there's a new file for all these component imports, `app/styles/_library/_components.scss`

Reply:
done